### PR TITLE
Add Google credentials setup instructions to README (#872)

### PR DESCRIPTION
**Pull request by danielsotirhos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +159,66 @@ end.
...
+    ```
+
+5. Move the saved json file to a safe location on your computer
+
+    ```
+    $ mkdir /Users/alice/.gcp
+    $ mv /Users/alice/Downloads/human-cell-atlas-travis-test-180b575fe2e6.json /Users/alice/.gcp/
+    ```
+    
+6. Edit your deployment's `environment.local` file, uncomment and modify the GOOGLE variables
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
6. Edit your deployment's `environment.local` file, uncomment and modify the `GOOGLE_…` variables
```

File: README.md [hunk 2]
```diff
@@ -157,7 +159,66 @@ end.
...
+
+    ```
+    $ vim /Users/alice/azul/deployments/alice.local/environment.local
+    
+    export GOOGLE_APPLICATION_CREDENTIALS="/Users/alice/.gcp/human-cell-atlas-travis-test-180b575fe2e6.json"
```
Comment (hannes-ucsc) [inline README.md]: Obfuscate.

File: README.md [hunk 3]
```diff
@@ -157,7 +159,66 @@ end.
...
+    export GOOGLE_PROJECT="human-cell-atlas-travis-test"
+    ```
+
+7. Create a symlink to your deployment `environment.local` file in other deployments as needed
```
Comment (hannes-ucsc) [inline README.md]: Try to not force people to use symlinks. They may already have symlinks, they may already have copies. Your instructions should work in both cases.

File: README.md [hunk 4]
```diff
@@ -157,7 +159,66 @@ end.
...
+    
+    Step 3:
+    [Create Key] -> JSON -> [Create] -> (Download & save json file) -> [Done]
+    ```
+
+5. Move the saved json file to a safe location on your computer
+
+    ```
+    $ mkdir /Users/alice/.gcp
+    $ mv /Users/alice/Downloads/human-cell-atlas-travis-test-180b575fe2e6.json /Users/alice/.gcp/
```
[the remark this answers is not in the document]
Comment (danielsotirhos) [reply README.md]: The hex string used in the example JSON filename was randomly generated and not taken from an actual JSON filename

File: README.md (hunk 2 above)
Comment (danielsotirhos) [reply README.md]: A full filename example was used to minimize the confusion that could arise if something like "{JSON-filename-here}" or "human-cell-atlas-travis-test....json" was used instead.

File: README.md [hunk 5]
```diff
@@ -157,7 +159,66 @@ end.
...
+
+3. Navigate to the `IAM & admin` page, locate your account in list, take note of the email address found in the `Member` column (eg. alice@example.com)
+
+   ```
+   Google Cloud Platform -> Navigation menu -> IAM & admin
+   ```
+
+4. Create a service account for project `human-cell-atlas-travis-test`
+
+    ```
```
[the remark this answers is not in the document]
Comment (hannes-ucsc) [reply README.md]: This is not what you use a pre-formatted section for. This should use either single backquotes as in 

```
`IAM & admin` -> `Service Accounts` -> `[Create Service Account]`
```

or emphasis as in 

```
*IAM & admin* -> *Service Accounts* -> *[Create Service Account]*
```

I think there is precedent for the latter in this document.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hannes-ucsc) [reply README.md]: Cool

**Pull request merged**